test/compiler-pending-utils.xspec

```diff
@@ -1,130 +1,128 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <x:description
-    xmlns:myv="http://example.org/ns/my/variable"
     xmlns:x="http://www.jenitennison.com/xslt/xspec"
     xmlns:xs="http://www.w3.org/2001/XMLSchema"
-    stylesheet="../../src/compiler/compile-xslt-tests.xsl"
+    stylesheet="../src/compiler/base/util/compiler-pending-utils.xsl"
     xslt-version="3.0">
 
-    <x:scenario label="Tests for 'stacked-explicit-pending' accumulator">
+    <x:helper stylesheet="../src/compiler/base/util/compiler-misc-utils.xsl" />
+
+    <x:scenario label="Tests for 'stacked-explicit-pending' accumulator" xml:base="end-to-end/cases/">
         <x:scenario label="Pending scenarios that use @pending and nest">
-            <x:variable name="myv:context" href="../end-to-end/cases/focus-vs-pending-attribute.xspec"/>
-            <x:call function="true"/>
+            <x:context href="focus-vs-pending-attribute.xspec"/>
```

Comment:
```suggestion
            <!-- Applying templates via <x:context> is not actually needed for checking
                accumulator values but makes the $x:context variable available. -->
            <x:context href="focus-vs-pending-attribute.xspec"/>
```

If someone wants to imitate this code in their own situation, applying templates might be noticeably more wasteful of time or memory than calling `true()` just to avoid an XSpec compiler error. If all they need are accumulator values, applying templates isn't necessary, and that's why I'm suggesting a comment to clarify that.

Reply:
Good point! Your clear comment is much appreciated.